fdm-app/app/components/custom/cultivation/list.tsx

```
@@ -0,0 +1,108 @@
+import { Button } from "@/components/ui/button"
+import { format } from "date-fns/format"
+import { Pencil, Trash2 } from "lucide-react"
+import { NavLink, useFetcher } from "react-router"
+import { LoadingSpinner } from "../loadingspinner"
+import type { Cultivation } from "./types"
+
+export function CultivationList({
+    cultivations,
+    harvests,
+}: { cultivations: Cultivation[]; harvests: any[] }) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)